### DISCUSSION TITLE
Fan Manual control

### DISCUSSION THREAD

**Discussion by HuguesGauthier** [Rem0o/FanControl.Releases#671] [Q&A] [answered]
I set my fan to manual control, and i set the slider to 0, but the fan still running a 1000 rpm !

![image](https://user-images.githubusercontent.com/8374362/165104533-56113135-ad80-4b7d-96f6-184e2471a5e6.png)

Any idea ?

thanks

**Comment by Rem0o** [comment]
Are you sure you got the RPM paired correctly?

**Comment by HuguesGauthier** [comment]
I did remove the pair speed sensor, and redo pair speed sensor

![image](https://user-images.githubusercontent.com/8374362/165110677-c7813e14-3c72-4804-8ce5-adbe2c0d1939.png)

now the icon is red.

**Reply by Rem0o** [reply]
Unhide the speed section, or use the automatic pairing.

**Comment by HuguesGauthier** [comment]
When i do 
![image](https://user-images.githubusercontent.com/8374362/165114434-e2b69a35-78e7-4dbb-815e-18c18328f838.png)

the fan never stop
![image](https://user-images.githubusercontent.com/8374362/165114530-906d1ac9-c55e-435c-83f2-0a1bad492ba0.png)

**Reply by Rem0o** [reply]
That means that header doesn't allow below at set threshold. Is that header a CPU header? That's fairly common. Like my CPU header on my mobo doesn't allow anything below 17%. It will show 0%, but it's effectively 17%.

**Reply by beyondthemind91an** [reply]
> That means that header doesn't allow below at set threshold. Is that header a CPU header? That's fairly common. Like my CPU header on my mobo doesn't allow anything below 17%. It will show 0%, but it's effectively 17%.

Hi Rem0o,

I have the same problem. I have a Asus Prime B560 plus. In the Ai Suite (fan xpert) I'm able to turn every fan completely off except for the CPU header, as it usually is. However, not in the FanControl software. That is, if I set a fan to 0% it still turns. I am however able to turn a fan completely off in FanControl if I first in Ai Suite turn off all the fans, close Ai Suite and then open FanControl. However, now I can't ramp the fans up above the lowest rpm that Ai Suite detected they can run. The program, FanControl, says it is sending a 100% speed signal, but the fans doesn't exceed around 500rpm. It is like I have to choose between:
500 to 2000rpm
Off/500rpm

Do you have any idea why this is happening?

**Comment by HuguesGauthier** [comment, marked as answer]
Ok the prob was that a uninstall Asus AI suite 3 after installing Fan control, and after that fan control lost control lol.
I reinstall AI suite 3, and fan control retake control.

![image](https://user-images.githubusercontent.com/8374362/165121579-99e9604a-bd40-44bb-9c54-65b8ec5bb86e.png)

**Reply by Rem0o** [reply]
You can now hide the speed section:
![image](https://user-images.githubusercontent.com/14118956/165121842-f578e72e-4605-43dd-b030-0c343da3bba5.png)

enjoy
